applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -16,6 +16,7 @@
 import subprocess
 import re
 import glob
+import getpass
```

Comment:
This is a new python tool, so this needs to be added to the requirements list.
It should probably go into /nrf/scripts/requirements-fixed.txt but please check with Thomas Stilwell or Sebastian Wezel.

Reply:
I will send a message:)

Reply:
I just checked, and it it a part of Python Standard Library, so I guess I dont need to (?)

Reply:
It is a part of Python Standard Library, so I dont need to